test/models/Capability/Model.test.ts

```diff
@@ -58,4 +59,11 @@ describe("Applicant model", () => {
 
     await expect(capability.save()).rejects.toThrow();
   });
+
+  it("has case-insensitive unique values for description", async () => {
```

Comment:
lo escribiria como: `should raise an error if adding existing case-insensitive description`